### PR TITLE
docs(devkit): add link to docs

### PR DESCRIPTION
**Pull request by juristr**
Small improvement to add a link for folks to the extending Nx part of the docs.

https://nx-dev-git-fork-juristr-docs-link-nx-devkit-nrwl.vercel.app/packages/devkit

### CODE REVIEW

File: docs/generated/manifests/packages.json [hunk 1]
```diff
@@ -566,12 +566,12 @@
     "githubRoot": "https://github.com/nrwl/nx/blob/master",
     "name": "devkit",
     "packageName": "@nx/devkit",
-    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline docs/generated/manifests/packages.json:569]:
```suggestion
    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

File: docs/generated/manifests/packages.json [hunk 2]
```diff
@@ -566,12 +566,12 @@
...
     "documents": {
       "/packages/devkit/documents/nx_devkit": {
         "id": "nx_devkit",
         "name": "Overview",
-        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline docs/generated/manifests/packages.json:574]:
```suggestion
        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs..",
```

File: docs/generated/packages-metadata.json [hunk 3]
```diff
@@ -558,12 +558,12 @@
     "source": "/packages/detox/src"
   },
   {
-    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline docs/generated/packages-metadata.json:561]:
```suggestion
    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

File: docs/generated/packages-metadata.json [hunk 4]
```diff
@@ -574,7 +574,7 @@
       {
         "id": "ngcli_adapter",
         "name": "Ng CLI Adapter",
-        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline docs/generated/packages-metadata.json:577]:
```suggestion
        "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

File: docs/packages.json [hunk 5]
```diff
@@ -93,7 +93,7 @@
   {
     "name": "devkit",
     "packageName": "devkit",
-    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline docs/packages.json:96]:
```suggestion
    "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

File: packages-legacy/devkit/package.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
   "name": "@nrwl/devkit",
   "version": "0.0.1",
   "private": false,
-  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline packages-legacy/devkit/package.json:5]:
```suggestion
  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

File: packages/devkit/package.json [hunk 7]
```diff
@@ -2,7 +2,7 @@
   "name": "@nx/devkit",
   "version": "0.0.1",
   "private": false,
-  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more.",
+  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit on our docs](https://nx.dev/extending-nx/intro/getting-started).",
```
Comment (meeroslav) [inline packages/devkit/package.json:5]:
```suggestion
  "description": "The Nx Devkit is used to customize Nx for different technologies and use cases. It contains many utility functions for reading and writing files, updating configuration, working with Abstract Syntax Trees(ASTs), and more. Learn more about [extending Nx by leveraging the Nx Devkit](https://nx.dev/extending-nx/intro/getting-started) on our docs.",
```

**Review by meeroslav — CHANGES_REQUESTED** [review]
Minor change for more semantical links

**Review by isaacplmann — APPROVED** [review]

**Review by meeroslav — APPROVED** [review]

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**